src/picasso-definition/components/mini-chart/mini-points.js

```diff
@@ -0,0 +1,69 @@
+import KEYS from '../../../constants/keys';
+import NUMBERS from '../../../constants/numbers';
+
+export default function createMiniChartPoints(chartModel) {
+  const ratio = NUMBERS.MINI_CHART.RATIO;
+  const padding = NUMBERS.MINI_CHART.PADDING; // Padding from the bottom right corner
+
+  // Padding, normalized to chart size
+  let px;
+  let py;
+
+  // Home state data view (corresponding the rect of the mini chart)
+  let xMin0;
+  let xMax0;
+  let yMin0;
+  let yMax0;
```

Comment:
Same as above.

For example, `homeState.min.x` reads a lot better than `xMin0`.